cpg_workflows/defaults.toml

```diff
@@ -1,14 +1,8 @@
-[workflow]
-# Prefix to find docker images referenced in `image_config_yaml_path`
-image_registry_prefix = 'australia-southeast1-docker.pkg.dev/cpg-common/images'
-
-# Prefix to find reference files referenced in `refdata_yaml_path`
-reference_prefix = 'gs://cpg-reference'
-
-# Template to build HTTP URLs matching the dataset_path of category
-# "web". Should be parametrised by namespace and dataset in Jinja format:
-web_url_template = 'https://{namespace}-web.populationgenomics.org.au/{dataset}'
+[images]
+dragmap = 'australia-southeast1-docker.pkg.dev/cpg-common/images/dragmap:1.3.0'
+cpg_workflows = 'australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_workflows:latest'
```

Comment:
In terms of reproducibility, is using the `latest` tag a bit dangerous here? Wouldn't the image config template contain an explicit tag for the newest version?

Reply:
We don't maintain `cpg_workflows` images through the images repo (it's build through the `production_pipelines` CI). Could move it there of course

Reply:
It would be too complicated though. I think I will just manage the tag here with bumpverison :)

Reply:
That's such an elegant solution! Thanks!